CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [5.3.2] - 2018-07-17
```

Comment:
Pode tirar isso, se vc tiver a ultima versao do Releasy, ele ja vai escrever isso pra vc

Reply:
@klzns wat

Reply:
A ultima versao do releasy atualiza o changelog.md automaticamente

Reply:
@klzns uhum entendi, eu to é surpreso